Gemfile.lock

```diff
@@ -45,7 +45,7 @@ GIT
 PATH
   remote: .
   specs:
-    chef (19.1.103)
+    chef (19.1.107)
```

Comment:
Wait why chef got bumped here? Is our update_version script working on PR merges?

Reply:
https://github.com/chef/chef/commit/3162d2c0930b6cc0ea07c6b1cf9ea4a39ce1b267#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731f everything but the top level chef version itself